docusaurus/docs/engineering/start-developing/boba-frontend.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 5
+---
+
+# Install BobaFrontend
+
+:::warning
+This codebase requires Node **16**. If you're using [Node Version Manager](https://github.com/nvm-sh/nvm), you can switch to version 16 by running `nvm use 16` in your terminal.
```

Comment:
Either up here or in the troubleshooting section I'd add a note about running yarn install using the correct version of node bc things get messed up otherwise